### PR TITLE
Add ability for admins to download files

### PR DESCRIPTION
**Pull request by cklosowski**
Adds a metabox for anyone who can 'edit_others_posts' that allows them to see all items attached to a download, and then offers some information regarding size and mime type, as well as a download link.

It addresses the concerns in #3002

### CODE REVIEW

File: includes/admin/upload-functions.php [hunk 1]
```diff
@@ -86,8 +87,8 @@ function edd_create_protection_files( $force = false, $method = false ) {
 			}
 		}
 		// Check for the files once per day
-		set_transient( 'edd_check_protection_files', true, 3600 * 24 );
-	}
+		//set_transient( 'edd_check_protection_files', true, 3600 * 24 );
```
Comment (pippinsplugins) [inline includes/admin/upload-functions.php]: This needs re-added :)

File: includes/admin/upload-functions.php [hunk 2]
```diff
@@ -52,7 +52,7 @@ function edd_change_downloads_upload_dir() {
  */
 
 function edd_create_protection_files( $force = false, $method = false ) {
-	if ( false === get_transient( 'edd_check_protection_files' ) || $force ) {
+	//if ( false === get_transient( 'edd_check_protection_files' ) || $force ) {
```
Comment (chriscct7) [inline includes/admin/upload-functions.php]: this needs to be undone as well

**Comment by rubengc** [comment]
No news here?

This is a useful feature to get files submitted from other users

**Comment by cklosowski** [comment]
@rubengc Not at this time, if you look at the main issue there was a LOT of discussion on the direction to go, with many different positions on how it should be handeled.

**Comment by rubengc** [comment]
Oh I see It and i added a suggestion

To me is a useful resource because I need manually revise this thought FTP, so this PR will save to me so much time

**Pull request closed without merging**